packages/office-addin-manifest/src/cli.ts

```diff
@@ -22,6 +22,13 @@ commander
 
 commander.command("validate <manifest-path>").action(commands.validate);
 
+commander
+  .command("export")
+  .option("-m, --manifest <manfest-path>", "Specify the location of the manifest file.  Default is './manifest.json'")
```

Comment:
The difference is that it becomes a required parameter without a default.  I noticed that on the other commands and wasn't sure why they didn't use a default instead of requiring it on the command line.  

If I was going to require and manual manifest argument what does that mean for the assets folder argument?